thetis/shallowwater_eq.py

```diff
@@ -160,11 +207,15 @@ class ShallowWaterTerm(Term):
     """
     def __init__(self, space,
                  bathymetry=None,
-                 nonlin=True):
+                 nonlin=True,
+                 wetting_and_drying=False,
+                 wd_alpha=0.5):
```

Comment:
Maybe a general design issue: There are now multiple places (perhaps 10?) where these default values appear.  Would it make sense to always pull them out of the model options, assuming those are available in this constructor?